oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,146 @@
+group of people participating in the project and helping to shape its direction will lead to a more
+vibrant platform ecosystem that fosters stronger innovative collaboration within the Open edX community,
+increased velocity that accelerates platform advancement, and increased capacity to own and maintain
+all aspects of the platform.
+
+Specification
+=============
+
+Core Contributor Roles
+----------------------
+
+There are many contributors to the Open edX platform that participate in a variety of ways - writing
+code for the Open edX software, translating the project, performing Quality Assurance tasks, participating
+in the Marketing working group, writing documentation, and more. At the time of writing we have identified
+the following roles: Project Manager, Translation Lead, Code Contributors, Forum Moderators, UI/UX Designers,
+Marketing Team Members, Documentation Writers, QA, and Product Managers.
+
+
+All CCs are granted some Rights to the project as part of participating - the ability to
+merge code, the final say on marketing copy, etc. In exchange for these Rights to contribute, we ask that
+CCs abide by their Responsibilities. For more on the CC roles, and Rights & Responsibilities, see:
+
+- `Core Contributor Role Definitions <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2759460357/Core+Contributor+Role+Definitions>`_
+- `Core Contributor Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2952003698/Core+Contributor+Rights+Responsibilities+excluding+code+contributors>`_
+- `Code Committer Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Rights+Responsibilities+for+Code+Contributors>`_
+
+Note that roles vary in what they need from members; the wiki will be the most up to date source
+of what each role entails and what types of commitments are expected.
+
+Core Contributor Program Administration
+---------------------------------------
+
+The Core Contributor Program is administered by The Center for Re-imagining Learning (tCRIL). One or more
+persons at tCRIL (the "Program administrator(s)") work with the community to make sure the program runs
+smoothly: this entails facilitating communication amongst community members, `selecting new members`_,
+and ensuring that existing members renew their commitment to the community and abide by the Code of Conduct.
+
+The tCRIL administrator(s) are the point people for making sure that needed agreements are signed - this includes
+a Program Agreement between the CC and tCRIL; for code contributors, an additional CLA; and for
+new entities joining the community, an Entity Contributor Agreement. We are additionally asking that all employers
+of CCs sign the `Revised Declaration of Commitment to the Core Contributor Program <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3216900524>`_
+to demonstrate that they are willing to engage with and commit resources to the Open edX project.
```

Comment:
Hmm. I wonder though if what a 3-person startup can offer is different from what a 30-person can? For example MITx pledges that Peter will write release notes for named releases. Should we say that's not enough?

Reply:
Yup, I'm not saying it can't be different -- but maybe just to have a process to review and approve it? Maybe something to make part of the review process of each new core contributor?

Reply:
What about something like, "We encourage a commitment of 20h/month from each of an organization's Core Contributor, however, we recognize that circumstances differ and a different commitment may be more appropriate. Before signing the agreement, please reach out to the Program Administrator to discuss."

?

And I should probably make an email list so people can actually contact the program administrators :)

Reply:
Sounds good to me :+1:  

Could this also be mentioned in the review of the candidature, so that can be considered as part of the review? Ie judge the skills, and also the exact commitment to the role being made?

Reply:
Yes, I can make that mention - do you mean just to consider their commitment holistically, or to make some level of commitment a requirement for being a CC?